src/modules/reserve-overview/components/UserInformation/index.tsx

```diff
@@ -122,7 +122,7 @@ export default function UserInformation({
     isReserveInEmode && isAssetStable(symbol) && formattedAvailableBorrows && !poolReserve.isFrozen;
 
   let isBorrowEnable =
-    !formattedAvailableBorrows || poolReserve.borrowingEnabled || !poolReserve.isFrozen;
+    !!formattedAvailableBorrows && poolReserve.borrowingEnabled && !poolReserve.isFrozen;
```

Comment:
formattedAvailableBorrows will always be a truthy string wont it? even in the case of '0'